Clarify some ext auth deployment modes docs (#1557)
solo-io/gloo: docs/content/gloo_routing/virtual_services/security/_index.md

more easily test the latency improvement gained by using the sidecar mode. Regardless of the deployment mode used, the values set
in `global.extensions.extAuth` will apply to both instances of Ext Auth.

> You can set which auth server that Envoy communicates with by updating your settings to point to the proper Ext Auth upstream.

Reviewer: nit, wording/brevity
```suggestion
You can set which auth server Envoy queries by updating your settings to use the proper Ext Auth upstream.
```

> On install, we manually create an upstream for each instance of Ext Auth that you have deployed; standalone deployments get an upstream named

Reviewer: use Gloo instead of we
```suggestion
On install, Gloo manually creates an upstream for each instance of Ext Auth that you have deployed; standalone deployments get an upstream named
```

> On install, we manually create an upstream for each instance of Ext Auth that you have deployed; standalone deployments get an upstream named

Reviewer: use Gloo instead of we
```suggestion
On install, Gloo manually creates an upstream for each instance of Ext Auth that you have deployed; standalone deployments get an upstream named
```

> `extauth`, and sidecar deployments get an upstream named `extauth-sidecar`. If you are deploying both standalone and sidecar, or 

Reviewer: nit, grammar
```suggestion
`extauth` and sidecar deployments get an upstream named `extauth-sidecar`. If you are deploying both standalone and sidecar, or 
```

just standalone, the default upstream name in your settings will be `extauth`. If you are deploying just the sidecar, the default upstream name in your 
settings will be `extauth-sidecar`.

Here is a look at a snippet of the `default` settings after a fresh install of Gloo, using the default Helm configuration; note the Ext Auth server ref specifically.

> ```

Reviewer: ```suggestion
```shell
```
